test/integration/console_scripts_test.py

```diff
@@ -1,20 +1,57 @@
 """Test the PUDL console scripts from within PyTest."""
-
-import importlib.metadata
+from pathlib import Path
 
 import pytest
+import sqlalchemy as sa
 
-# Obtain a list of all deployed entry point scripts to test:
-PUDL_SCRIPTS = [
-    ep.name
-    for ep in importlib.metadata.entry_points(group="console_scripts")
-    if ep.value.startswith("pudl")
-]
 
+@pytest.mark.parametrize(
+    "command",
+    [
+        # Force the download of one small partition from Zenodo
+        "pudl_datastore --dataset ferc60 --bypass-local-cache --partition year=2020",
+        # Force the download of one small partition from GCS
+        "pudl_datastore -d ferc60 --bypass-local-cache --gcs-cache-path gs://zenodo-cache.catalyst.coop --partition year=2020",
+        # Exercise the datastore validation code
+        "pudl_datastore -d ferc60 --validate --partition year=2020",
+        # Ensure that all data source partitions are legible
+        "pudl_datastore --list-partitions --gcs-cache-path gs://zenodo-cache.catalyst.coop",
+    ],
+)
+@pytest.mark.script_launch_mode("inprocess")
+def test_pudl_datastore(script_runner, command: str):
+    """CLI tests specific to the pudl_datastore script."""
+    runner_args = command.split(" ")
+    ret = script_runner.run(runner_args, print_result=True)
+    assert ret.success
 
-@pytest.mark.parametrize("script_name", PUDL_SCRIPTS)
+
+@pytest.mark.parametrize(
+    "command,filename",
+    [
+        (
+            "pudl_service_territories --entity-type ba -y 2022 --no-dissolve -o ",
+            "balancing_authority_geometry.parquet",
+        ),
+        (
+            "pudl_service_territories --entity-type util -y 2022 --dissolve -o ",
+            "utility_geometry_dissolved.parquet",
```

Comment:
With just a single year of data, we can actually test both ba/util and dissolved/undissolved geometries in a few seconds.